Makefile

```diff
@@ -471,8 +438,14 @@ pr:
 
 .PHONY: golangci-lint
 golangci-lint:
-	@hash golangci-lint > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
```

Comment:
Maybe we can keep those checks but call `make prepare-tools` if needed ?

Reply:
Do you mean check the tools at the call sites or check them under `prepare-tools`?